### DISCUSSION TITLE
Ferramenta de comunicação

### DISCUSSION THREAD

**Discussion by mazulo** [apyb/comunidade#4] [Geral]
Na últimas duas edições da Python Brasil, conversamos brevemente sobre como se dá a comunicação da APyB e a comunidade. Vários pontos foram levantados, inclusive sobre a questão das listas de email não serem tão utilizadas. Nessa issue vamos discutir um pouco mais sobre esse tema, bem como estudar alternativas.

**Comment by mazulo** [comment]
Estou testando duas ferramentas:

- https://revolt.chat/ (uma alternativa open source para o Discord)
- https://app.element.io/ (outra alternativa de chat)
- https://docs.github.com/en/discussions (recomendação @rougeth)
- https://github.com/discourse/discourse

Em breve mais comentários

**Reply by rennerocha** [reply]
O Discourse é utilizado pela PSF (https://discuss.python.org/) e dependendo como é configurado, as pessoas podem interagir com ele através de e-mail (quase como em uma lista de e-mail). Eu gosto da interface e de como é possível criar várias discussões distintas. Uma vantagem é ser código-aberto e poder ser self-hosted, o que nos livra de estar trancados em uma plataforma fechada.

**Reply by rennerocha** [reply]
Também temos usado o Discourse no hackerspace LHC (https://discourse.lhc.net.br/). Acho uma alternativa legal, inclusive para disponibilizar reportes da diretoria (podemos inclusive restringir acesso a certos items para alguns usuarios se quisermos)

**Reply by rougeth** [reply]
Humberto chegou a criar uma instância do Discourse (https://apyb.python.org.br/) mas não chegamos a testar de verdade. Eu gosto muito da ideia do Discourse (API disponível resolveria vários problemas). Dito isso, sou contra mantermos self-hosted.

**Reply by mazulo** [reply]
@rougeth que doidera!! Eu tava **completamente** por fora de que tínhamos isso! @rougeth você saberia onde isso tá hospedado? Se é pago ou algo assim?

**Reply by rougeth** [reply]
@mazulo deve estar na Configr, lembro de estarmos hospedando sim, mas não cheguei a fazer nada no lado da infra.

**Reply by mazulo** [reply]
@rougeth beleza, valeu. Vou dar uma olhada lá pra saber em que pé está isso

**Reply by rennerocha** [reply]
> Humberto chegou a criar uma instância do Discourse (https://apyb.python.org.br/) mas não chegamos a testar de verdade. Eu gosto muito da ideia do Discourse (API disponível resolveria vários problemas). Dito isso, sou contra mantermos self-hosted.

Acho que "poder" ser self-hosted é uma feature interessante em uma ferramenta para a comunidade, já que não ficamos sujeitos a ter que sair de algum serviço por que ele encerrou (por exemplo, se um dia o Github Discussions for encerrado, perderíamos isso).

Agora precisamos manter nós mesmos? Não. Podemos contratar uma empresa que faça a manutenção do serviço de pé para nós, assim não temos que nos preocupar em manter a infraestrutura e só utilizar o serviço. Se essa empresa não estiver fazendo um bom serviço, podemos procurar outra por exemplo. Em https://www.discourse.org/pricing existem planos para associações non-profit com desconto por exemplo.

Acho que precisamos normalizar mais o pagar por serviços para terceiros. E dar preferências para soluções de código aberto.

**Reply by gabubellon** [reply]
@rennerocha 
> Acho que precisamos normalizar mais o pagar por serviços para terceiros
Essa frase é perfeito e vale pra outras parteas (Marketing, Financeiro, Juridico etc)

>  E dar preferências para soluções de código aberto.
Eu gosto disso, mas acho que em alguns momentos é mais prático resolver com coisas fechadas e que já são solução de mercado, mas acho sim que devemos olhar para coisas  abertas

Uma proposta: se custo for viável podemos testar o https://www.discourse.org/pricing para associações non-profit inicialmente dentro da diretoria e conselhos, acho que vale minimanete uma tarefa de todo  para a médio prazo (tem coisas bem mais prioritárias)

**Comment by rougeth** [comment]
@mazulo chegou a considerar o [Github Discussions](https://docs.github.com/en/discussions)? Está mais para o lado da lista de email do que de chats/discord

Ótima ideia ter criado esse repositório! 👏

**Comment by mazulo** [comment]
@rougeth ainda não, na realidade eu nem conhecia! Ótima ideia, vou adicionar na lista e começar a dar uma olhada

**Comment by cuducos** [comment]
As ferramentas listadas acho que dão conta da interação entre que faz parte da associação — o que é ótimo.

 No tópico _comunicação_ acho que podemos considerar também um canal onde a diretoria presta contas periodicamente (algo como um relatório mensal do que foi feito, o foco do próximo mês, etc) — por exemplo, isso pode ser uma newsletter, um post num blog, um post no Medium etc.

A ideia é aproximar a diretoria, e reforçar os canais de comunicação da associação etc.

**Reply by rennerocha** [reply]
Mesmo sem as pessoas usarem muito e-mail hoje em dia, eu acho que uma newsletter (por email e divulgada em outros canais como Telegram, Twitter, etc) com um relatório da diretoria mostrando o que foi feito em um período e o que está nos planos, seria muito interessante. Acho que isso ajudaria aos associados não esquecerem que são associados e daria uma noção de que existe trabalho sendo feito.

**Reply by gabubellon** [reply]
Complementando o @rennerocha, acho super válido a news+ redes, inclusive acho que a Diretoria de Mkt deve estar ligada em todas as novas redes afim de fazer buzz nelas e trazer as pessoas para os canais melhores para gente

**Comment by rougeth** [comment]
No ponto do @cuducos, usar issues do Github foi uma ótima experiência pra org da pybr2021:
- Fácil de ver o que estava sendo feito.
- Fácil de resgatar o que foi feito.
- Fácil de pesquisar por algo específico.
- Maioria das pessoas estavam acostumadas com a ferramenta
- Para quem prefere email, como eu :D, ainda é uma opção

**Reply by gabubellon** [reply]
@rougeth você fiz usar as issues para prestar contas e afins ??
Eu gosto de uma ideia de uma newsletter igual a OK faz, porque pode ser acessível por mais pessoas.
Outro ponto `Maioria das pessoas estavam acostumadas com a ferramenta` não sei dizer se a maioria das pessoas estariam familizadas com issues, isso pode ser uma verdade dentro da nossa bolha

**Reply by rennerocha** [reply]
Eu acho que existe um certo preciosismo em buscar uma ferramenta que a maioria das pessoas está acostumada. Toda ferramenta vai ter uma curva de aprendizado para um certo grupo de pessoas e isso não é necessariamente ruim. Não podemos ser tão paternalistas e achar que as pessoas não vão conseguir aprender uma coisa nova caso seja de interessa delas de participar mais ativamente da associação.

**Reply by rougeth** [reply]
@gabubellon

> @rougeth você fiz usar as issues para prestar contas e afins ??

Eu acho que se o trabalho é feito de forma pública, a prestação de contas já é feita por definição. Pra mim, isso é muito mais importante do que só ter a newsletter e posts em redes sociais. Diretorias futuras também se beneficiariam da documentação gerada.

> isso pode ser uma verdade dentro da nossa bolha

Boa. Foi uma verdade para a organização da Python Brasil, não sabemos se isso se aplica as pessoas associadas.

@rennerocha 

> Eu acho que existe um certo preciosismo em buscar uma ferramenta que a maioria das pessoas está acostumada.

Meu ponto nem é curva de aprendizagem. Na minha opinião, escolher uma ferramenta que não faz parte do que as pessoas já usam, tem um risco muito grande de diminuir ainda mais a pouca interação entre as pessoas associadas. Hoje, pouca gente participa na lista (muitos fatores contribuem para isso, inclusive o meio), mas todo mundo usa email, não é um app novo que só vai ser usado, de vez em nunca, para se comunicar com a APyB.

**Reply by rennerocha** [reply]
@rougeth na verdade eu acho que estamos numa situação boa para escolher a melhor ferramenta que atenda aos objetivos de comunicação, já que como não temos interação nenhuma com a ferramenta atual qualquer outra coisa pode ser melhor :-)

Também sabemos que nenhuma ferramenta é boa o suficiente, sempre vai ter alguém que vai ter mais dificuldades ou não vai querer utilizar ela (por exemplo, se a comunicação principal ficar num servidor Discord, é mto provável que eu não participe por não eu gostar da ferramenta).

Acessibilidade é importante, queremos mais participação. Mas uma hora vamos ter que definir uma estratégia de comunicação e interação sabendo que teremos pessoas associadas que podem não se sentir motivadas a participar de acordo com as ferramentas escolhidas. Dito isso, estou gostando desse DIscussions do Github!

**Reply by rennerocha** [reply]
> Eu acho que se o trabalho é feito de forma pública, a prestação de contas já é feita por definição. Pra mim, isso é muito mais importante do que só ter a newsletter e posts em redes sociais. Diretorias futuras também se beneficiariam da documentação gerada.

Sim e não (na minha opinião). É importante (e necessário) o trabalho ser feito de forma pública, mas organizar os dados e fazer um sumário do que aconteceu também ajuda as pessoas a entenderem o contexto geral investindo pouco tempo para isso. A partir desse sumário, a pessoa pode então buscar no histórico público do que foi feito os detalhes de como as coisas foram feitas.

**Reply by gabubellon** [reply]
@rougeth eu acho que além de forma publica, tem que que existem propaganda, porque por esse lado temos muito dados publico que niguém conhece.
@rennerocha sobre a ferramenta, entendo seu ponto, mas eu por exemplo faço parte de um grupo onde tem pessoas que não sabem usar um GDocs ou um call no meeting direito e temos que ficar ensinado (ensinar não é e nunca será um problema) porque a vivencia dessa pessoa é Facebook e Whatsapp.
Eu acho DIscussionsé algo completamente nichado, pois atende quem tem e conhece rasoávelmente github, mas sou a favor de uma ferramente parecidade e mais aberta (Discorse ???) e de constantre treinamento
Eu sempre brinco que se eu for em um meetup de python e ensinar uma pessoa a fazer "Hello Word" pela primeira vez to satisfeito, então porque não term esse mesmo foco em fazer alguém "achar" e usar a ferramenta que escolhermos mas de maneira mais abrangente ??

**Reply by gabubellon** [reply]
Ah , eu to adorando isso aqui também !!

**Reply by cuducos** [reply]
> Eu acho que se o trabalho é feito de forma pública, a prestação de contas já é feita por definição.

Discordo. Resumo em uma analogia: uma coisa é dado granular, outra é dado agregado. Ambos são importantes, e dependendo do fim, um é melhor que o outro.

Claro que com o dado granular tu chega no dado agregado — mas demanda tempo e esforço. Minha sugestão de canal aberto para interação (dado granular) **mais** prestação de conta periódica (dado agregado) tem com objetivo aproximar as pessoas da APyB **sem** demandar mais tempo e esforço dessas pessoas.

**Reply by mazulo** [reply]
@cuducos qual canal aberto para interação (dado granular) dentre os mencionados (ou não mencionados também) na sua opinião seriam interessantes?

**Reply by cuducos** [reply]
Na minha opinião, dos que eu conheço, só o Discourse e essa aba de Discussions do GitHub (ou mesmo o grupo de emails), @mazulo.

As discussões em plataformas como o Discord tendem a ser efêmeras: uma mensagem/tópico não é facilmente linkado ou compartilhado, as discussões não são buscáveis (ou indexadas pelo Google). Isso tende a fazer muita coisa se perder no tempo, ter papos cíclicos etc.

**Reply by rougeth** [reply]
Eu me expressei mal, eu concordo que a prestação de conta periódica é importante. Eu não estava em sync com o que vocês estavam falando. Minha visão é de quem está um pouco mais próximo da APyB por conta das pybrs (interação é alta entre org e apyb) e por isso estava valorizando mais um dos lados (dados granular :). Mas concordo que ambos são importantes sim.
